src/auth/constants.ts

```diff
@@ -6,6 +6,8 @@ export const bcryptConstants = {
 
 export const serviceConstants = { name: 'Recipe App' };
 
+export const numberOf2faRecoveryTokens = 8;
+
 export const strategyNameConstants = {
```

Comment:
hmm, I think we should change the naming convention for these values to Upper case letters. So:
```suggestion
export const SERVICE_CONSTANTS = { NAME: 'Recipe App' };

export const NUMBER_OF_2FA_RECOVERY_TOKENS = 8;

export const STRATEGY_NAME_CONSTANTS = {
```
etc.

also not sure about e.g. `STRATEGY_NAME_CONSTANTS`. Maybe it should be just a `STRATEGY` so that in other parts of the code we can use e.g. `STRATEGY.LOCAL`